### PR TITLE
JavadMotallebi.NeatDownloadManager version 1.3

### PR DESCRIPTION
**Pull request by One11Onee**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27356)

### CODE REVIEW

**Comment by QingSaYa** [comment]
Validation-Completed

**Comment by QingSaYa** [comment]
@msftbit Validation-Completed

File: manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JavadMotallebi.NeatDownloadManager
+PackageVersion: 1.3
```
Comment (ItzLevvie) [inline manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.3.24
```

File: manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: JavadMotallebi.NeatDownloadManager
+PackageVersion: 1.3
```
Comment (ItzLevvie) [inline manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.yaml:5]:
```suggestion
PackageVersion: 1.3.24
```

File: manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JavadMotallebi.NeatDownloadManager
+PackageVersion: 1.3
```
Comment (ItzLevvie) [inline manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.installer.yaml:5]:
```suggestion
PackageVersion: 1.3.24
```

File: manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JavadMotallebi.NeatDownloadManager
+PackageVersion: 1.3
+PackageLocale: en-US
+Publisher: Javad Motallebi
+PackageName: Neat Download Manager
+License: NDM
+ShortDescription: Neat Download Manager(NDM/NeatDM) Setup
```
Comment (ItzLevvie) [inline manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Neat Download Manager is a free Internet Download Manager.
```

File: manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JavadMotallebi.NeatDownloadManager
+PackageVersion: 1.3
+PackageLocale: en-US
+Publisher: Javad Motallebi
+PackageName: Neat Download Manager
+License: NDM
```
Comment (ItzLevvie) [inline manifests/j/JavadMotallebi/NeatDownloadManager/1.3/JavadMotallebi.NeatDownloadManager.locale.en-US.yaml:9]:
```suggestion
License: Freeware
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**